![](mila-medical-logo-small.png)

# Mila Medical Reading Group

A group to discuss methods related to genomics, imaging, or clinical data. Enumerating all successful and impactful applications of ML/DL in medicine.

Organizer: [Joseph Paul Cohen](https://josephpcohen.com)

Join the mailing list here: https://groups.google.com/forum/#!forum/mila-medical

| Date | Speaker | Title | Reference |
|---|--|--|----|
| 4/8/2019 | | | |
| 4/1/2019 | | | |
| 3/25/2019 | | | |
| 3/18/2019 | | | |
| 3/11/2019 | Thomas Rademaker |  <details><summary>Attack and defence in cellular decision-making: lessons from machine learning</summary><p>Abstract: Machine learning algorithms are sensitive to meaningless (or "adversarial") perturbations. This is reminiscent of cellular decision-making where ligands (called "antagonists") prevent correct signalling, like in early immune recognition. We draw a formal analogy between neural networks used in machine learning and models of cellular decision-making (adaptive proofreading). We apply attacks from machine learning to simple decision-making models, and show explicitly the correspondence to antagonism by weakly bound ligands. Such antagonism is absent in more nonlinear models, which inspired us to implement a biomimetic defence in neural networks filtering out adversarial perturbations. We then apply a gradient-descent approach from machine learning to different cellular decision-making models, and we reveal the existence of two regimes characterized by the presence or absence of a critical point. The critical point causes the strongest antagonists to lie close to the threshold. This is validated in the loss landscapes of robust neural networks and cellular decision-making models, and observed experimentally for immune cells. For both regimes, we explain how associated defence mechanisms shape the geometry of the loss landscape, and why different adversarial attacks are effective in different regimes. Our work connects evolved cellular decision-making to machine learning, and motivates the design of a general theory of adversarial perturbations, both for in vivo and in silico systems.</p></details> | [Paper](https://arxiv.org/abs/1807.04270) |
| 3/4/2019 |Geneviève Boucher | |  |
| 2/25/2019 | - | Cancelled for Comp4Med |  |
| 2/18/2019 | Camille Rochefort-Boulanger |  <details><summary>Comprehensive functional genomic resource and integrative model for the human brain</summary><p>Abstract: In this study, they build a gene regulatory network (GRN) for the brain, linking transcription factors, enhancers, and target genes from merging of the QTLs, generalized element-activity correlations, and Hi-C data. They integrate this GRN to a deep Boltzmann machine, by manually adding links between the visible nodes of the network given the relations from the GRN, to perform disease prediction for psychiatric disorders.</p></details> | [Paper](http://science.sciencemag.org/content/362/6420/eaat8464) |
| 2/11/2019 | Louis Collins | MRI basics: short presentation, questions + answers |  |
| 2/4/2019 | Baptiste Goujaud | Use of the adversarial training in medical imaging across examples |  |
| 1/28/2019 | Alex Zabbal | DL applications for Neuroscience |  |
| 1/21/2019 | Mandana Samiei | Training Medical Image Analysis Systems like Radiologists | [Paper](https://arxiv.org/abs/1805.10884) |
| 1/14/2019 | Martin Weiss and Paul Bertin | Gene Graph Convolutions |  |
| 12/12/2018 | Mohammed AlQuraishi | End-to-End Differentiable Learning of Protein Structure | [Paper](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3239970) |
| 11/28/2018 | Georgy Derevyanko | Predicting protein complexes: classical approaches and deep learning |  |
| 11/14/2018 | Julien Cohen | Connecting MRI physics and A.I. to advance neuroimaging | [Slides](slides/20181114_MILA_seminar.pdf) |
| 11/7/2018 | Georgy Derevyanko | Learning Protein Structure with a Differentiable Simulator |  |
| 10/31/2018 | Suzanne Tamang | Clinical tools: A survey of Suzanne Tamang's work |  |
| 10/24/2018 | Martin Weiss | A Survey of Mobile Computing for the Visually Impaired | [Paper](https://arxiv.org/abs/1811.10120) |
| 10/17/2018 | Paul Bertin | Uncertainty measures for Multiple Sclerosis Lesion Segmentation | [Paper](https://arxiv.org/abs/1808.01200) |
| 10/10/2018 | Abdou Mousas | Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations|  |
| 10/3/2018 | Vincent Frappier | Lessons from Natural Language Inference in the Clinical Domain |  |
| 9/12/2018 | Assya Trofimov | dna2vec: Consistent vector representations of variable-length k-mers | [Paper](https://arxiv.org/abs/1701.06279)  |
| 9/5/2018 | Lucie Blais | Medication adherence as a predictor of morbidity and mortality in patients with chronic respiratory diseases |  |
| 8/22/2018 | Francis Dutil | DeepSEA Predicting effects of noncoding variants with deep learning–based sequence model |  |
| 8/8/2018 | Vincent Frappier | Recomendation of ICD codes literature review |  |
| 8/1/2018 | Martin Weiss | Syntax-Directed Variational Autoencoder for Structured Data |  |
| 6/27/2018 | Martin Weiss | Recurrent Neural Networks for Multivariate Time Series with Missing Values |  |
| 6/20/2018 | Vincent Frappier | Exploring sequence binding space using coevolution and structural tertiary motifs |  |
| 6/13/2018 | Joseph Paul Cohen | A survey of clinical data successes using machine learning | [Slides](https://docs.google.com/presentation/d/1hmJKGmreVzfGCjtcgy3aAFpuZHRNymN4vCBGpvfL0S0/edit) |
| 5/30/2018 | Georgy Derevyanko | Prediction of toxicity from chemical composition of a drug |  |
| 5/16/2018 | Georgy Derevyanko | End2end differentiable protein structure learning |  |
| 4/25/2018 | Everyone | Project updates |  |
| 4/18/2018 | Everyone | Project updates |  |
| 4/11/2018 | Georgy Derevyanko | Inverse Statistical Physics of Protein Sequences:  A Key Issues Review |  |
| 4/4/2018 | Joseph Cohen | Generative adversarial networks uncover epidermal regulators and predict single cell perturbations |  |
| 3/28/2018 | Everyone | Slide Karaoke |  |
| 3/21/2018 | Martin Weiss | Network-based machine learning and graph theory algorithms for precision oncology |  |
| 3/14/2018 | Everyone | Project updates |  |
| 3/7/2018 | Francis Dutil | So you are presenting Convolutional Networks on Graphs for Learning Molecular Fingerprints | [Paper](https://arxiv.org/abs/1509.09292) |
| 2/28/2018 | Christopher Beckham | Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules |  |
| 2/21/2018 | Christopher Beckham & Geneviève Boucher | The cornucopia of meaningful leads: Applying deep adversarial autoencoders for new molecule development in oncology [Paper](http://www.oncotarget.com/index.php?journal=oncotarget&page=article&op=view&path%5B0%5D=14073&path%5B1%5D=44886) & Near-optimal probabilistic RNA-seq quantification [Paper](https://www.nature.com/articles/nbt.3519) |  |
| 2/14/2018 | Everyone | Project updates |  |
