![](mila-medical-logo-small.png)

# Mila Medical Reading Group

A group to discuss methods related to genomics, imaging, or clinical data. Enumerating all successful and impactful applications of ML/DL in medicine.

Organizer: [Joseph Paul Cohen](https://josephpcohen.com)

Join the mailing list here: https://groups.google.com/forum/#!forum/mila-medical

| Date | Speaker | Title | Reference |
|---|--|--|----|
| 4/29/2019 | | | |
| 4/22/2019 | Georgy Derevyanko | <details><summary>Protein-protein docking using learned three-dimensional representations </summary><p>Abstract: Proteins are the cogs of cellular machinery. Understanding how these molecules interact and predicting structures of their complexes is unavoidable step we need to pass to understand emergence of the phenotype of an organism from its genotype. Rigid-body docking approaches the problem by assuming two proteins rigid and sampling rotations and translations of the ligand with respect to the receptor. The key idea, that allows the sampling to be performed in a reasonable timeframe is representation of the interaction energy between the two proteins as correlations of 3d protein representations. However, these representations themselves are either pretrained or handcrafted. In this work we present an algorithm that allows learning protein representations for docking in the end-to-end fashion.</p></details> | |
| 4/15/2019 |Phil Novosad |Self-ensembling for domain adaptation: application to neuroanatomical segmentation in MRI | |
| 4/8/2019 | Maxime Thibault | <details><summary>Prediction of the next medication order to assist prescription verification by pharmacists in a health care center</summary><p> In North American hospitals, pharmacists review all medication orders to ensure adequate medication use. As pharmacists move to a practice model involving direct patient care, the activity of drug order verification could benefit from a certain degree of automation to free up resources to perform clinical tasks. Potential applications of machine learning include detecting atypical orders and focusing pharmacist attention on unusual patterns instead of verifying every single order. We prototyped a machine learning model to predict the next prescribed drug during hospitalization, and we compared model predictions with pharmacist ratings on a sample of orders. This presentation will discuss health system pharmacy and the data it generates, the clinical problem we faced, how we developed a model that shows potential at helping pharmacists detect atypical orders, and how the predictions of this model correlate with pharmacist ratings.</p></details>| |
| 4/1/2019 | Becks Simpson | | |
| 3/25/2019 | | | |
| 3/18/2019 | Frederic Leblond |  <details><summary>Towards a Multimodal Spectroscopy Imaging Plateform for Intraoperative Tissue Characterization</summary><p>Abstract: We are developing multimodal intraoperative imaging tools based on an imaging platform combining Raman, fluorescence and diffuse reflectance (multi-spectral and SFDI) with the objective of gathering large spectral datasets across organ sites/pathologies and patients for the development of classification models using machine learning techniques. The combinaison of Raman, fluorescence and diffuse reflectance spectroscopic images offers the potential to provide a breadth of molecular features that can be employed for surface as well as for depth-resolved tissue characterisation. Using a versatile wide-field spectroscopic imaging plateform, databases of tissue-specific spectroscopic information can be acquired for future artificial intelligence applications. Here we will present recent results relating with the development of each imaging modality as well as progress towards the development of the multimodal imaging plateform and the wide-field intraoperative imaging probes to be used for data gathering in the scope of clinical studies. </p></details> | |
| 3/11/2019 | Thomas Rademaker |  <details><summary>Attack and defence in cellular decision-making: lessons from machine learning</summary><p>Abstract: Machine learning algorithms are sensitive to meaningless (or "adversarial") perturbations. This is reminiscent of cellular decision-making where ligands (called "antagonists") prevent correct signalling, like in early immune recognition. We draw a formal analogy between neural networks used in machine learning and models of cellular decision-making (adaptive proofreading). We apply attacks from machine learning to simple decision-making models, and show explicitly the correspondence to antagonism by weakly bound ligands. Such antagonism is absent in more nonlinear models, which inspired us to implement a biomimetic defence in neural networks filtering out adversarial perturbations. We then apply a gradient-descent approach from machine learning to different cellular decision-making models, and we reveal the existence of two regimes characterized by the presence or absence of a critical point. The critical point causes the strongest antagonists to lie close to the threshold. This is validated in the loss landscapes of robust neural networks and cellular decision-making models, and observed experimentally for immune cells. For both regimes, we explain how associated defence mechanisms shape the geometry of the loss landscape, and why different adversarial attacks are effective in different regimes. Our work connects evolved cellular decision-making to machine learning, and motivates the design of a general theory of adversarial perturbations, both for in vivo and in silico systems.</p></details> | [Paper](https://arxiv.org/abs/1807.04270) |
| 3/4/2019 |Geneviève Boucher |  <details><summary>DeepSynergy: predicting anti-cancer drug synergy with Deep Learning</summary><p>Abstract: In this paper, the authors applied a deep learning approach for predicting the synergy of drug combinations. DeepSynergy uses compound information as well as genomic information as inputs to a regression task in order to find specific drug combinations that have maximal efficacy in specific genomic context, in their case in given cancer cell lines.</p></details>  | [Paper](https://academic.oup.com/bioinformatics/article/34/9/1538/4747884) |
| 2/25/2019 | - | Cancelled for [AI for Health Event at UdeM](https://www.meetup.com/AIForHealth/events/259166623/) |  |
| 2/18/2019 | Camille Rochefort-Boulanger |  <details><summary>Comprehensive functional genomic resource and integrative model for the human brain</summary><p>Abstract: In this study, they build a gene regulatory network (GRN) for the brain, linking transcription factors, enhancers, and target genes from merging of the QTLs, generalized element-activity correlations, and Hi-C data. They integrate this GRN to a deep Boltzmann machine, by manually adding links between the visible nodes of the network given the relations from the GRN, to perform disease prediction for psychiatric disorders.</p></details> | [Paper](http://science.sciencemag.org/content/362/6420/eaat8464) |
| 2/11/2019 | Louis Collins | MRI basics: short presentation, questions + answers |  |
| 2/4/2019 | Baptiste Goujaud | Use of the adversarial training in medical imaging across examples |  |
| 1/28/2019 | Alex Zabbal | DL applications for Neuroscience |  |
| 1/21/2019 | Mandana Samiei | Training Medical Image Analysis Systems like Radiologists | [Paper](https://arxiv.org/abs/1805.10884) |
| 1/14/2019 | Martin Weiss and Paul Bertin | Gene Graph Convolutions |  |
| 12/12/2018 | Mohammed AlQuraishi | End-to-End Differentiable Learning of Protein Structure | [Paper](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3239970) |
| 11/28/2018 | Georgy Derevyanko | Predicting protein complexes: classical approaches and deep learning |  |
| 11/14/2018 | Julien Cohen | Connecting MRI physics and A.I. to advance neuroimaging | [Slides](slides/20181114_MILA_seminar.pdf) [gSlides](https://docs.google.com/presentation/d/1IN0lLS8lV_vSEwwyI6Vrb43HCc6ahtjY9wof_-vB0x4/edit) |
| 11/7/2018 | Georgy Derevyanko | Learning Protein Structure with a Differentiable Simulator |  |
| 10/31/2018 | Suzanne Tamang | Clinical tools: A survey of Suzanne Tamang's work |  |
| 10/24/2018 | Martin Weiss | A Survey of Mobile Computing for the Visually Impaired | [Paper](https://arxiv.org/abs/1811.10120) |
| 10/17/2018 | Paul Bertin | Uncertainty measures for Multiple Sclerosis Lesion Segmentation | [Paper](https://arxiv.org/abs/1808.01200) |
| 10/10/2018 | Abdou Mousas | Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations|  |
| 10/3/2018 | Vincent Frappier | Lessons from Natural Language Inference in the Clinical Domain | [Slides](slides/MILA_Medical_NLP_EHR.pdf) |
| 9/12/2018 | Assya Trofimov | dna2vec: Consistent vector representations of variable-length k-mers | [Paper](https://arxiv.org/abs/1701.06279)  |
| 9/5/2018 | Lucie Blais | Medication adherence as a predictor of morbidity and mortality in patients with chronic respiratory diseases |  |
| 8/22/2018 | Francis Dutil | DeepSEA Predicting effects of noncoding variants with deep learning–based sequence model |  |
| 8/8/2018 | Vincent Frappier | Recomendation of ICD codes literature review |  |
| 8/1/2018 | Martin Weiss | Syntax-Directed Variational Autoencoder for Structured Data | [gSlides](https://docs.google.com/presentation/d/1acNguM5q2Y0clg74dxFpnP3HMT9HhD6ayzhOnmbLmv4/edit) [Video](https://www.youtube.com/watch?v=6HqIk-PxpCo) |
| 6/27/2018 | Martin Weiss | Recurrent Neural Networks for Multivariate Time Series with Missing Values |  |
| 6/20/2018 | Vincent Frappier | Exploring sequence binding space using coevolution and structural tertiary motifs |  |
| 6/13/2018 | Joseph Paul Cohen | A survey of clinical data successes using machine learning | [Slides](https://docs.google.com/presentation/d/1hmJKGmreVzfGCjtcgy3aAFpuZHRNymN4vCBGpvfL0S0/edit) |
| 5/30/2018 | Georgy Derevyanko | Prediction of toxicity from chemical composition of a drug |  |
| 5/16/2018 | Georgy Derevyanko | End2end differentiable protein structure learning |  |
| 4/25/2018 | Everyone | Project updates |  |
| 4/18/2018 | Everyone | Project updates |  |
| 4/11/2018 | Georgy Derevyanko | Inverse Statistical Physics of Protein Sequences:  A Key Issues Review |  |
| 4/4/2018 | Joseph Cohen | Generative adversarial networks uncover epidermal regulators and predict single cell perturbations | [Slides](https://docs.google.com/presentation/d/1rS6kPXosNbSOWOiI2C8E18KWo20XEPubx6_VURYCco0/edit) |
| 3/28/2018 | Everyone | Slide Karaoke |  |
| 3/21/2018 | Martin Weiss | Network-based machine learning and graph theory algorithms for precision oncology |  |
| 3/14/2018 | Everyone | Project updates |  |
| 3/7/2018 | Francis Dutil | So you are presenting Convolutional Networks on Graphs for Learning Molecular Fingerprints | [Paper](https://arxiv.org/abs/1509.09292) |
| 2/28/2018 | Christopher Beckham | Automatic Chemical Design Using a Data-Driven Continuous Representation of Molecules |  |
| 2/21/2018 | Christopher Beckham & Geneviève Boucher | The cornucopia of meaningful leads: Applying deep adversarial autoencoders for new molecule development in oncology [Paper](http://www.oncotarget.com/index.php?journal=oncotarget&page=article&op=view&path%5B0%5D=14073&path%5B1%5D=44886) & Near-optimal probabilistic RNA-seq quantification [Paper](https://www.nature.com/articles/nbt.3519) |  |
| 2/14/2018 | Everyone | Project updates |  |
